#! /bin/sh -e
## 40_help_browser.dpatch by Petter Reinholdtsen <pere@hungry.com>
##
## All lines beginning with `## DP:' are a description of the patch.
##
## DP: Use sensible-browser instead of dillo for displaying the help text.

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch $patch_opts -p0 < $0;;
       -unpatch) patch $patch_opts -p0 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0
@DPATCH@
Index: src/preferences
===================================================================
RCS file: /cvsroot/roadmap/roadmap/src/preferences,v
retrieving revision 1.6
diff -u -3 -p -u -r1.6 preferences
--- src/preferences	29 Nov 2004 07:02:09 -0000	1.6
+++ src/preferences	26 Feb 2005 12:15:26 -0000
@@ -12,6 +12,7 @@ GPS.Color: red
 GPS.Foreground: black
 GPS.Source: gpsd://localhost
 GPS.Timeout: 10
+Help.Browser: sensible-browser
 Highlight.Background: yellow
 Highlight.Duration: 10
 Highlight.Thickness: 4
